doxygen/examples/H5.format.html

```diff
@@ -11276,7 +11287,7 @@ <h4><a name="DatatypeMessage">IV.A.2.d. The Datatype Message</a></h4>
                         </tr>
 
                         <tr>
-                          <td colspan="4"><br />Base Type<br /><br /></td>
+                          <td colspan="4"><br />Parent Type Message<br /><br /></td>
```

Comment:
To me, "base type" seems more appropriate since a variable-length type is similar to a container around some other type.

Reply:
The documentation generally seems to use 'base type' and 'parent type' interchangeably. Enums and arrays are also described as having 'parent types' - would it be best to standardize all of these cases to refer to 'base types'? The field in the shared datatype structure is named `parent`, but that may not matter since it's internal.

Reply:
I definitely think it would be good to standardize on a term, whether that's 'base type' or 'parent type'. The library definitely uses the two interchangeably fairly often, but I've personally always found 'parent type' to be confusing when used to refer to the type that a container type (vlen, array, compound) contains. The header comment for `H5Tvlen_create` even says "Create a new variable-length datatype based on the specified BASE_TYPE." and the parameter is `base_id`.

@derobins @fortnern Any opinions on this?